ekg.cabal

```diff
@@ -27,14 +27,12 @@ tested-with:         GHC == 8.0.1, GHC == 7.10.1, GHC == 7.8.4, GHC == 7.6.3
 
 library
   exposed-modules:
+    Paths_ekg
```

Comment:
That fails on Travis with ghc 7.6.3 (https://travis-ci.org/tibbe/ekg/jobs/155201949). Not sure why.